joss/paper.md

```diff
@@ -30,40 +30,40 @@ Consequently, these populations undergo evolution [@pennock2007models], and can
 
 Vast amounts of bioinformatics-oriented phylogenetics software is also available.
 These programs' purposes typically include
-- inferring phylogenies from extant organisms (and sometimes fossils) [@challa2019phylogenetic]
-- sampling phylogenies from theoretical models of population and species dynamics [@stadler2011simulating]
-- cross-referencing phylogenies with other data (e.g., spatial species distributions) [@emerson2008phylogenetic]
-- analyzing and manipulating tree structures [@smith2020treedist;@sand2014tqdist;@sukumaran2010dendropy;@cock2009biopython]
+- inferring phylogenies from extant organisms (and sometimes fossils) [@challa2019phylogenetic],
+- sampling phylogenies from theoretical models of population and species dynamics [@stadler2011simulating],
+- cross-referencing phylogenies with other data (e.g., spatial species distributions) [@emerson2008phylogenetic], and
+- analyzing and manipulating tree structures [@smith2020treedist;@sand2014tqdist;@sukumaran2010dendropy;@cock2009biopython].
 
 Phylotrack overlaps with these goals only in that it also provides tree statistic implementations. We chose to include this feature to facilitate fast during-simulation calculations of these metrics.
-Notably, the problem of tracking a phylogeny within an agent-based program is substantially different from the more traditional problem of reconstructing a phylogeny
-(indeed, users new to working with recorded phylogenies should refer to the Phylotrackpy documentation for notes on subtle structural differences from reconstructed phylogenies).
+Notably, the problem of tracking a phylogeny within an agent-based program is substantially different from the more traditional problem of reconstructing a phylogeny.
+(Indeed, users new to working with recorded phylogenies should refer to the Phylotrackpy documentation for notes on subtle structural differences from reconstructed phylogenies.)
 
-Phylotrack has already facilitated several research projects.
+Phylotrack has contributed to a variety of published research.
 Phylotracklib has been integrated into packages such as Modular Agent-Based Evolver (MABE) 2.0 [@bohm2019mabe], Symbulation [@vostinarSpatialStructureCan2019], and even a fork of the Avida digital evolution platform [@ofria2004avida;@dolsonInterpretingTapeLife2020].
-These applications have facilitated research on open-ended evolution [@dolsonMODESToolboxMeasurements2019], the origin of endosymbiosis [@johnsonEndosymbiosisBustInfluence2022a], the importance of phylogenetic diversity for machine learning via evolutionary computation [@hernandez2022can;@shahbandegan2022untangling], and more.
+Applications include research on open-ended evolution [@dolsonMODESToolboxMeasurements2019], the origin of endosymbiosis [@johnsonEndosymbiosisBustInfluence2022a], the importance of phylogenetic diversity for machine learning via evolutionary computation [@hernandez2022can;@shahbandegan2022untangling], and more.
```

Comment:
Ah, I see how this was unclear - I meant that Phylotracklib got incorporated into those platforms, and then the platforms were able to support the research mentioned here. i.e. the "applications" are MABE, Symbulation, etc. How about: "Through these integrations, Phylotracklib has enabled research..."

Reply:
I was trying to go for "applications" in the general sense of having been used instead of the "software applications" sense.

We could phrase as "research applications include projects investigating"

Reply:
mostly trying to cut down on our count of the word "facilitating" and phrase active voice

Reply:
I think we want this to be in the past tense, since we've just scratched the surface so far and these are just examples of what has been done so far, not a list of possible future applications. "Phylotracklib has enabled research..." is still activate voice

Reply:
Coming around to "has enabled"

I think we were throwing around enable on the topic sentence of the paragraph? Should be fine as long as it reads not too close together

Reply:
Okay, sounds good!